server/service/teams.go

```diff
@@ -294,6 +337,13 @@ func deleteTeamUsersEndpoint(ctx context.Context, request interface{}, svc fleet
 		return teamResponse{Err: err}, nil
 	}
 
+	// mask the API token of integrations
```

Comment:
As far as I could see those are the only places where we return the team's config (e.g. in team users we return the teams the user is part of, but the team config is not included - let me know if you think I've missed any!)